llama_index/embeddings/base.py

```diff
@@ -132,6 +132,60 @@ async def aget_query_embedding(self, query: str) -> Embedding:
             )
         return query_embedding
 
+    def _get_image_embedding(self, img_file_path: ImageType) -> Embedding:
```

Comment:
why do we need to make these changes? we already have the `MultiModalEmbedding` class